safely_report/survey/xlsform_functions.py

```diff
@@ -29,3 +33,33 @@ def _selected_at(cls, choice_array: list[str], index: int) -> str:
             return choice_array[index]
         except IndexError:
             return ""
+
+    @classmethod
+    def _str_to_int(cls, value: str):
+        return int(value)
+
+    @classmethod
+    def _pulldata(cls, column, default):
+        """
+        Assumes that all data is pulled from the respondent roster table
+
+        Parameters
+        ----------
+        column: the field you want to retrieve
+        default: a default value in case the field is not available, or the
+            user is an Enumerator
+
+        Returns
+        -------
+        field value if available, default otherwise
```

Comment:
I suggest we use consistent [NumPy style](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), like so:

```
Parameters
----------
column: str
    Name of the field you want to retrieve
default: str
    Default value in case the field is not available,
    or the user is an Enumerator

Returns
-------
str
    Field value if available, default otherwise
```